### PR TITLE
Add type annotation to simpleMapExpr

### PR DESCRIPTION
**Pull request by FBroy**
# Description

Add type annotation to simpleMapExpr.

## Deletes Source Branch?

Yes:    Work is done, the branch can be safely deleted.

## How Many Approvals?

* 2 Approvals (Normal features and bugs)

### CODE REVIEW

**Review by HectorPeeters — CHANGES_REQUESTED** [review]
Looks good to me but it would be nice to have some tests for this as well

**Review by HectorPeeters — APPROVED** [review]
Now we know this functionality works as expected!

File: src/typeInference/annotateAST.ts [hunk 1]
```diff
@@ -235,6 +236,9 @@ export function annotate(ast: IAST, staticContext: StaticContext): SequenceType
 			return annotateCastableOperator(ast);
 
 		// Maps
+		case 'simpleMapExpr':
+			astHelper.getChildren(ast, 'pathExpr').map((c) => annotate(c, staticContext));
```
Comment (MTimmerije) [inline src/typeInference/annotateAST.ts:240]: nice, mapping over the map

**Review by MTimmerije — APPROVED** [review]
Liked, favourited, subscribed and approved.

**Pull request merged**